ui/static/js/confirm_handler.js

```diff
@@ -1,18 +1,4 @@
 class ConfirmHandler {
-    executeRequest(url, redirectURL) {
```

Comment:
This sounds so generic, but was only used in one place. It's new home in `FeedHandler` is easier to locate, IMO.

Reply:
This is used in many places across the application, not only to unsubscribe feeds, this why it's generic. See https://github.com/miniflux/miniflux/search?q=data-confirm%3D%22true%22&unscoped_q=data-confirm%3D%22true%22

Reply:
Thanks a lot, I was too trigger happy here. This part of the patch is reverted (but `FeedHandler` is still added, to handle the keyboard shortcut without any confirmation step).